operations/experimental/migrate-cf-mysql-to-pxc.yml

```diff
@@ -3,9 +3,17 @@
   path: /releases/-
   value:
     name: pxc
-    sha1: bd784bb93ffeff62cdec6aeb62ba0fea4eba6f41
-    url: https://bosh.io/d/github.com/cloudfoundry-incubator/pxc-release?v=0.4.0
-    version: 0.4.0
+    sha1: 4dd970acd8fb059e73d9922adb9cc40206334882
+    url: https://bosh.io/d/github.com/cloudfoundry-incubator/pxc-release?v=0.6.0
+    version: 0.6.0
+
+- type: replace
+  path: /releases/-
```

Comment:
Can we change this to a update-or-append operator? I'm worried about the case where an operator has added os-conf-release in a runtime-config or something. I thing it would need to be something like
```
- type: replace
  path: /releases/name=os-conf?
  value:
  ...
```

Reply:
Sounds good. Pushed a commit to update that.